build/workbox.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const path = require('path')
+const swBuild = require('workbox-build')
+const config = require('./workbox.config.json')
+const buildPrefix = '_gh_pages/'
+
+const updateUrl = (manifestEntries) => manifestEntries.map((entry) => {
+  if (entry.url.startsWith(buildPrefix)) {
+    const regex = new RegExp(buildPrefix, 'g')
+    entry.url = path.normalize(entry.url.replace(regex, ''))
```

Comment:
I don't think this will fix the issue. `path.posix` should be used, but I'll try it later and see :)

Reply:
@Lyricalz Please remove the workaround since the issue shouldn't happen. I have opened an issue upstream.

Reply:
@XhmikosR done :)